app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    cmd, origin, dupe = command.split()
+
+    if cmd != "cp" or origin == dupe:
+        return
+    try:
+        with open(origin, "r") as source, open(dupe, "w") as copy:
+            copy.write(source.read())
```

Comment:
Code Efficiency: Consider checking if the `dupe` file already exists before attempting to write to it. Overwriting an existing file without warning can lead to data loss.